src/backend/distributed/transaction/remote_transaction.c

```diff
@@ -303,17 +303,8 @@ FinishRemoteTransactionCommit(MultiConnection *connection)
 
 		if (transaction->transactionState == REMOTE_TRANS_1PC_COMMITTING)
 		{
-			if (transaction->transactionCritical)
-			{
-				ereport(WARNING, (errmsg("failed to commit critical transaction "
-										 "on %s:%d, metadata is likely out of sync",
-										 connection->hostname, connection->port)));
-			}
-			else
-			{
-				ereport(WARNING, (errmsg("failed to commit transaction on %s:%d",
-										 connection->hostname, connection->port)));
-			}
+			ereport(WARNING, (errmsg("failed to commit transaction on %s:%d",
```

Comment:
I wonder, are we supposed to set transactionFailed here? Not sure we want to change it right now, but it's a little odd that we don't.

Reply:
We seem to handle it in the `HandleRemoteTransactionResultError()` called few lines before this change.

Reply:
Thanks! Missed that.